### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by okaris** [tencent-ailab/IP-Adapter#120]
Firstly, I'd like to extend my heartfelt appreciation for the exceptional models and pipelines you've developed. They are truly remarkable and have greatly aided my work.

I am interested in adapting the recently released SDXL face model for a different training objective. To better tailor the model to my needs, I'd love to understand the dataset and loss function you've utilized in more detail.

While the examples provided in the notebook are astonishing in their quality, I've encountered challenges in reproducing similar results with different image inputs. I wonder if this may be due to the model being trained on synthetic face images.

Your insights on the training process would be invaluable to me. If you could take the time to elaborate on this, I would be extremely grateful.

Thank you once again for your exceptional work and looking forward to your response.

**Comment by xiaohu2015** [comment]
@okaris for current version, we trained face model using the same  architecture as IP-Adapter-plus: we use the cropped face as image condition and original image as the target.

For the training data, you can refer to this: https://github.com/tencent-ailab/IP-Adapter/issues/74 
For the preprocessing of face image, you can refer to this: https://github.com/tencent-ailab/IP-Adapter/issues/54

**Comment by eezywu** [comment]
@xiaohu2015 Have you tried removing the background & hair of the face image? Could you share more about what you tried to improve the face consistency? 

Looking forward to your response. Thanks!

**Comment by xiaohu2015** [comment]
> @xiaohu2015 Have you tried removing the background & hair of the face image? Could you share more about what you tried to improve the face consistency?
> 
> Looking forward to your response. Thanks!

yes, removing the background & hair of the face image is helpful

**Comment by bent1e** [comment]
how to removing the hair?When my face has long bangs, the image generated by the faceid
 also has the same bangs, which is not what I want
